scripts/lint-grafana-dashboard.mjs

```diff
@@ -10,6 +10,7 @@ import fs from "node:fs";
 @typescript-eslint/no-unsafe-assignment,
 @typescript-eslint/explicit-function-return-type,
 @typescript-eslint/naming-convention,
+quotes,
```

Comment:
There is a conflict between eslint and prettier if you try to defined a string such as `'job=~"$beacon_job|beacon"'`, it won't let you escape double quotes `\"` and eslint will complain if single quote or backtick is used.